app/assets/stylesheets/common/components/more-topics.scss

```diff
@@ -95,3 +92,12 @@
     max-width: 150px;
   }
 }
+
+
+#main-outlet .regular {
+  @media screen and (min-width: 550px) {
+    .more-topics__container .nav li .btn {
+      padding: 0.75em 0.65em;
+    }
+  }
+}
```

Comment:
Selector is later in group, because page specificity for ``.regular`` is needed to apply specific styling to topic view only.